src/libraries/System.Runtime.InteropServices.JavaScript/src/System/Runtime/InteropServices/JavaScript/Legacy/Runtime.cs

```diff
@@ -9,6 +9,11 @@ namespace System.Runtime.InteropServices.JavaScript
     [Obsolete]
     public static class Runtime
     {
+        static unsafe Runtime () {
+            // HACK: For some reason the linker trims this class even if I set all the attributes to disable trimming.
```

Comment:
with the code in `JavaScriptExports` you probably would not have to do this.

Reply:
Sadly I still need it. I feel like something is wrong with the linker.

Reply:
mark the method `public` because we protect it as `[DynamicDependency(DynamicallyAccessedMemberTypes.PublicMethods`